histoprint/formatter.py

```diff
@@ -384,7 +384,7 @@ def __init__(
         if self.summary:
             # Make room for a summary at the bottom
             self.hist_lines -= 4
-        elif any([len(l) > 0 for l in self.labels]):
+        elif any([len(lab) > 0 for lab in self.labels]):
```

Comment:
```suggestion
        elif any(len(lab) > 0 for lab in self.labels):
```

This will be faster, too.